elements/pfe-avatar/demo/index.html

```diff
@@ -22,15 +22,15 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/vue/2.5.2/vue.min.js"></script>
 
     <!-- uncomment the es5-adapter if you're using the umd version -->
-    <script src="/examples/node_modules/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js"></script>
-    <script src="/examples/node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
-    <script src="/examples/node_modules/requirejs/require.js"></script>
-    
-    <script>require([
-      '../dist/pfe-avatar.umd.js'
-    ])</script>
+    <!-- <script src="/examples/node_modules/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js"></script> -->
```

Comment:
Why change the demo page rendering in this PR?  I think it's nice to keep it the other way too for the IE11 validation.